specifyweb/specify/views.py

```diff
@@ -1367,3 +1367,9 @@ def parse_locality_set_foreground(collection, column_headers: List[str], data: L
         return 422, errors
 
     return 200, parsed
+
+
+#check for institution at login
+def get_institution(request):
+    institution = spmodels.Institution.objects.first()
+    return http.JsonResponse(institution if institution else None, safe=False)
```

Comment:
If we go with a route `is_new_user` instead, this function could simply calculate a boolean with something like
```
is_new_user = len(spmodels.Institution.objects.all()) == 0
```